### PR TITLE
Missing Spike Tasks Quest items

### PR DESCRIPTION
**Pull request by cbrm**

### CODE REVIEW

File: data/items/items.xml [hunk 1]
```diff
@@ -25661,6 +25661,10 @@
 		<attribute key="description" value="This sword was forged with the enthusiasm and dreams of the Rookgaardians." />
 	</item>
 	<item id="18554" name="crystal rubbish" />
+	<item id="18559" article="an" name="adventurer's stone">
+		<attribute key="weight" value="3" />
```
Comment (Luckey1729) [inline data/items/items.xml]: 0.3 oz is expressed as "30" for the weight attribute key.

Comment (cbrm) [inline data/items/items.xml]: right, fixed now.

**Comment by ranisalt** [comment]
Squash the commits

**Comment by cbrm** [comment]
And by 'squash' you mean...?

**Comment by ranisalt** [comment]
By squash I mean... squash. Google for it or read:
https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

Here, another PR where commits needed to be squashed:
https://github.com/otland/forgottenserver/pull/1370

**Comment by WibbenZ** [comment]
I had no luck with that tutorial or other things I found on google.
This is what I used (from Ninja) and it worked :p 
```
git checkout master
git reset --soft HEAD~9
git add -A
git commit -m "Fix report widget"
git push -f
```

In this case, change ```HEAD~9``` to ```HEAD~3``` since you have 3 commits to squash.

**Comment by cbrm** [comment]
Seems I messed up my branch while trying that, I'll start all over again.

**Pull request closed without merging**